src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -287,10 +287,6 @@ pub struct Args {
     /// The name of the admin role in Frontegg.
     #[clap(long, env = "FRONTEGG_ADMIN_ROLE", requires = "frontegg-tenant")]
     frontegg_admin_role: Option<String>,
-    /// A common string prefix that is expected to be present at the beginning
-    /// of all Frontegg passwords.
-    #[clap(long, env = "FRONTEGG_PASSWORD_PREFIX", requires = "frontegg-tenant")]
-    frontegg_password_prefix: Option<String>,
```

Comment:
Does this need to be coordinated with the cloud team before a release?

Reply:
Indeed, the Nightly caught it, and @arusahni has a fix https://github.com/MaterializeInc/materialize/commit/d49a4981ab5e04f6a4201b54c58859344c4876a0